### PR TITLE
Add support for PDP joysticks

### PR DESCRIPTION
**Pull request by idorobotics**
Added a config file to support [PDP type](https://pdp.com/) joystick controllers.

### CODE REVIEW

**Review by ahcorde — APPROVED** [review]
LGTM, @clalancette any objections?

**Comment by clalancette** [comment]
> LGTM, @clalancette any objections?

No objections, though I think a link inside of the config file to the type of joystick would be valuable (otherwise I would have no idea what PDP is, other than the PDP-11).

**Comment by idorobotics** [comment]
@clalancette do you mean adding a comment at the top of the file linking to https://pdp.com/ PDP website?

**Comment by clalancette** [comment]
> @clalancette do you mean adding a comment at the top of the file linking to https://pdp.com/ PDP website?

That or another place that actually shows what a "PDP" controller is, and why it differs from the more standard ones.

**Comment by idorobotics** [comment]
> > @clalancette do you mean adding a comment at the top of the file linking to https://pdp.com/ PDP website?
> 
> That or another place that actually shows what a "PDP" controller is, and why it differs from the more standard ones.

Sure, updated to add more info about PDP.

File: config/pdp.config.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#Configuration for PDP joystick controllers. See https://pdp.com/
```
Comment (clalancette) [inline config/pdp.config.yaml]:
```suggestion
# Configuration for PDP joystick controllers. See https://pdp.com/
```

**Review by clalancette — CHANGES_REQUESTED** [review]
One minor formatting fix, then I'm happy with this.

**Review by clalancette — APPROVED** [review]
Thanks!  Looks good to me.

**Pull request merged**